contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-rule.y

```diff
@@ -475,6 +475,72 @@ tsql_AlterLoginStmt:
 				}
 		;
 
+tsql_enable_disable_trigger:
+			tsql_enable_disable TRIGGER tsql_trigger_list ON relation_expr
+				{
+					AlterTableCmd *n1;
+					AlterTableStmt *n3 = makeNode(AlterTableStmt);
+					ListCell *lc;
+
+					foreach(lc, $3)
+					{
+						List *lst = lfirst_node(List, lc);
+						n1 = makeNode(AlterTableCmd);
+
+						if ($1)
+						{
+							n1->subtype = AT_EnableTrig;
+						}
+						else
+						{
+							n1->subtype = AT_DisableTrig;
+						}
+						
+						if (list_length(lst) > 1)
+						{
+							n1->name = strcat(((String *)list_nth(lst,0))->sval, ".");
+							n1->name = strcat(n1->name, ((String *)list_nth(lst,1))->sval);
```

Comment:
I think we can use `psprintf` here for these two directly, `strcat` is not safe to use.

Reply:
replaced `strcat` with `psprintf`